paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewState.kt

```diff
@@ -70,6 +70,7 @@ internal sealed class CustomerSheetViewState(
     }
 
     data class AddPaymentMethod(
+        val formViewModel: FormViewModel? = null,
```

Comment:
This doesn’t feel right. What we should probably do instead is to pass the `FormViewModel`’s state into `AddPaymentMethod`.